.eslintrc

```diff
@@ -0,0 +1,78 @@
+{
+    "env": {
+        "commonjs": true,
+        "node": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "arrow-spacing": [
```

Comment:
I deleted some of the rules and added no-useless-concat. Could you check each rule if they are ok? We can delete ones that are unnecessary.